docs/ko/docs/tutorial/extra-models.md

```diff
@@ -0,0 +1,223 @@
+# 추가 모델
+
+지난 예제에 이어서, 연관된 모델을 여러개 갖는 것은 흔한 일입니다.
+
+특히 사용자 모델의 경우에 그러한데, 왜냐하면:
+
+* **입력 모델** 은 비밀번호를 가져야 합니다.
+* **출력 모델** 은 비밀번호를 가지면 안됩니다.
+* **데이터베이스 모델** 은 해시처리된 비밀번호를 가질 것입니다.
+
+/// danger | 위험
+
+절대 사용자의 비밀번호를 평문으로 저장하지 마세요. 항상 이후에 검증 가능한 "안전한 해시(secure hash)"로 저장하세요.
+
+만약 이게 무엇인지 모르겠다면, [security chapters](security/simple-oauth2.md#password-hashing){.internal-link target=_blank}.에서 비밀번호 해시에 대해 배울 수 있습니다.
+
+///
+
+## 다중 모델
+
+아래는 비밀번호 필드와 해당 필드가 사용되는 위치를 포함하여, 각 모델들이 어떤 형태를 가질 수 있는지 전반적인 예시입니다:
+
+{* ../../docs_src/extra_models/tutorial001_py310.py hl[7,9,14,20,22,27:28,31:33,38:39] *}
+
+
+/// info | 정보
+
+Pydantic v1에서는 해당 메서드가 `.dict()`로 불렸으며, Pydantic v2에서는 `.model_dump()`로 이름이 변경되었습니다. `.dict()`는 여전히 지원되지만 더 이상 권장되지 않습니다.
+
+여기에서 사용하는 예제는 Pydantic v1과의 호환성을 위해 `.dict()`를 사용하지만, Pydantic v2를 사용할 수 있다면 `.model_dump()`를 사용하는 것이 좋습니다.
+
+///
+
+### `**user_in.dict()` 에 대하여
+
+#### Pydantic의 `.dict()`
+
+`user_in`은 Pydantic 모델 클래스인 `UserIn`입니다.
+
+Pydantic 모델은 모델 데이터를 포함한 `dict`를 반환하는 `.dict()` 메서드를 제공합니다.
+
+따라서, 다음과 같이 Pydantic 객체 `user_in`을 생성할 수 있습니다:
+
+```Python
+user_in = UserIn(username="john", password="secret", email="john.doe@example.com")
+```
+
+그 다음, 다음과 같이 호출합니다:
+
+```Python
+user_dict = user_in.dict()
+```
+
+이제 변수 `user_dict`에 데이터가 포함된 `dict`를 가지게 됩니다(이는 Pydantic 모델 객체가 아닌 `dict`입니다).
+
+그리고 다음과 같이 호출하면:
+
+```Python
+print(user_dict)
+```
+
+Python의 `dict`가 다음과 같이 출력됩니다:
+
+```Python
+{
+    'username': 'john',
+    'password': 'secret',
+    'email': 'john.doe@example.com',
+    'full_name': None,
+}
+```
+
+#### `dict` 언패킹(Unpacking)
+
+`user_dict`와 같은 `dict`를 함수(또는 클래스)에 `**user_dict`로 전달하면, Python은 이를 "언팩(unpack)"합니다. 이 과정에서 `user_dict`의 키와 값을 각각 키-값 인자로 직접 전달합니다.
+
+따라서, 위에서 생성한 `user_dict`를 사용하여 다음과 같이 작성하면:
+
+```Python
+UserInDB(**user_dict)
+```
+
+다음과 같은 결과를 생성합니다:
+
+```Python
+UserInDB(
+    username="john",
+    password="secret",
+    email="john.doe@example.com",
+    full_name=None,
+)
+```
+
+혹은 더 정확히 말하자면, `user_dict`를 직접 사용하여, 미래에 어떤 내용이 포함되더라도 다음과 같이 생성합니다:
+
+```Python
+UserInDB(
+    username = user_dict["username"],
+    password = user_dict["password"],
+    email = user_dict["email"],
+    full_name = user_dict["full_name"],
+)
+```
+
+#### 다른 Pydantic 모델의 데이터를 활용한 Pydantic 모델 생성
+
+위의 예제에서 `user_in.dict()`로부터 `user_dict`를 생성한 것처럼, 아래 코드는:
+
+```Python
+user_dict = user_in.dict()
+UserInDB(**user_dict)
+```
+
+다음과 동일합니다:
+
+```Python
+UserInDB(**user_in.dict())
+```
+
+...왜냐하면 `user_in.dict()`는 `dict`이며, 이를 `**`로 Python이 "언팩(unpack)"하도록 하여 `UserInDB`에 전달하기 때문입니다.
```

Comment:
```suggestion
...왜냐하면 `user_in.dict()`는 `dict`이며, 이를 `**`로 Python이 "언팩"하도록 하여 `UserInDB`에 전달하기 때문입니다.
```

Reply:
@sodaMelon
Thank you very much for reviewing my PR.

Would you mind explaining why you prefer not to mix English with Korean in your commit messages? 
For your reference, the term was recently updated from "unwrap" to "unpack," and I intentionally included the English version to emphasize that change.

Reply:
@timothy-jeong 
here are the refs ( same content)
- https://fastapi.tiangolo.com/ja/tutorial/extra-models 
- https://fastapi.tiangolo.com/zh/tutorial/extra-models

Reply:
@sodaMelon 
what an excellent reference! I always find it valuable to review translation examples from other languages.

However, in our Korean translated documents, we have several similar instances, such as:

like 
- [경로 매개변수](https://fastapi.tiangolo.com/ko/tutorial/path-params/#_17)
- [폼 모델](https://fastapi.tiangolo.com/ko/tutorial/request-form-models/#_3)
- [교차 출처 리소스공유](https://fastapi.tiangolo.com/ko/tutorial/cors/#_3) 
...

Additionally, incorporating English alongside Korean can actually enhance clarity in conveying the intended meaning.

I appreciate your review; however, if the rationale is "fixed repetition of words to make clear," I find it difficult to agree with that approach, especially since other Korean translation documents follow this convention.

Reply:
I think the problem is that the Korean rule was written too long ago.🤔(just written May, 2021. This is common with localization issues)

🤔🤔🤔If you need to sync with older docs, your way is great

Reply:
@sodaMelon 
yeah you can think about the korean rule is written too long ago. 
but in the reply section, as you know, we are talk about our standard.

It might come off as a bit blunt, but I think if we're going to change a convention, we should discuss it in our discussion forum.